tools/map/build_terrain_grid.py

```
@@ -0,0 +1,803 @@
+        cell_lab[c] = k
+    src, lab = list(cell_lab), list(cell_lab.values())
+    _, raw, _ = multi_dijkstra(field, src)
+    tbl = np.array(lab + [-1], dtype=np.int32)
+    seat_label = tbl[np.where(raw >= 0, raw, len(lab))].astype(np.int32)
+    roads, _ = build_roads(terrain, pts, hubs)
+    ford_list = fords(terrain, roads)
+
+    # 郡 인접의 도하 판정. 이동 단위가 郡이므로 여기만 실제 경로를 뽑는다.
+    land_field = cost_field(terrain, LAND_COST)
+    for (x, y) in pts:
+        x, y = int(x), int(y)
```

Ruff (0.16.1)

[warning] `for` loop variable `x` overwritten by assignment target

(PLW2901)